### ISSUE TITLE
CodeBlockWriter is not a constructor in v17.0.0

### ISSUE THREAD

**Issue by SferaDev** [dsherret/ts-simple-ast#1357]
**Describe the bug**

Version: 17.0.0

After upgrading to v17 I began receiving the following error when our tests attempted to run ts-morph:

```
TypeError: CodeBlockWriter is not a constructor
 ❯ ProjectContext.createWriter packages/codegen/node_modules/ts-morph/dist/ts-morph.js:20007:16
    20005|     createWriter() {
    20006|         const indentationText = this.manipulationSettings.getIndentati…
    20007|         return new CodeBlockWriter({
       |                ^
    20008|             newLine: this.manipulationSettings.getNewLineKindAsString(…
    20009|             indentNumberOfSpaces: indentationText === exports.Indentat…
 ❯ SourceFile._getWriter packages/codegen/node_modules/ts-morph/dist/ts-morph.js:3987:30
 ❯ SourceFile._getWriterWithChildIndentation packages/codegen/node_modules/ts-morph/dist/ts-morph.js:3969:29
 ❯ getNewText packages/codegen/node_modules/ts-morph/dist/ts-morph.js:2489:31
 ❯ insertIntoBracesOrSourceFile packages/codegen/node_modules/ts-morph/dist/ts-morph.js:2481:21
 ❯ insertIntoBracesOrSourceFileWithGetChildren packages/codegen/node_modules/ts-morph/dist/ts-morph.js:2513:5
 ❯ SourceFile._insertChildren packages/codegen/node_modules/ts-morph/dist/ts-morph.js:11468:20
 ❯ SourceFile.insertImportDeclarations packages/codegen/node_modules/ts-morph/dist/ts-morph.js:9161:25
 ❯ SourceFile.addImportDeclarations packages/codegen/node_modules/ts-morph/dist/ts-morph.js:9140:25
 ❯ SourceFile.addImportDeclaration packages/codegen/node_modules/ts-morph/dist/ts-morph.js:9
```

**To Reproduce**

Need to investigate further, opening for awareness.

**Comment by zivarah** [comment]
I'm also running into this. It looks like this line in the distributed ts-morph.js file:

`var CodeBlockWriter = require('code-block-writer');`

Is returning an object with a default property, but the ts-morph code is expecting the class to be returned directly:

```
CodeBlockWriter
{default: ƒ, __esModule: true}
default:
class CodeBlockWriter
__esModule:
true
[[Prototype]]:
Object
```

This feels very related to [esModuleInterop](https://www.typescriptlang.org/tsconfig#esModuleInterop), but unsure if that's actually in play here or not.

**Comment by dsherret** [comment]
That's weird. It should have compiled with esModuleInterop, but it seems like the output doesn't have that and I believe my tests didn't catch it because they're run through ts-node. I'll look into this right away.

Edit: Caused by the upgrade of rollup from v2 to v3.3

**Comment by dsherret** [comment]
Sorry about that. Should be fixed in 17.0.1 by reverting rollup from v3 to v2
